### PR TITLE
python3.pkgs.linien: init at 0.5.3

### PR DESCRIPTION
**Pull request by doronbehar**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
@ofborg build python3Packages.microscope

Tests fail for me on macos 10.15. Possibly should mark as broken?

File: pkgs/development/python-modules/pyrp3/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  '';
+
+  postBuild = ''
+    python setup.py lib_build
+  '';
+  preInstall = ''
+    python setup.py lib_install
+  '';
+  # No tests
+  doCheck = false;
```
Comment (risicle) [inline pkgs/development/python-modules/pyrp3/default.nix:32]: `pythonImportsCheck` though?

File: pkgs/development/python-modules/misoc/default.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  propagatedBuildInputs = [
+    pyserial
+    asyncserial
+    jinja2
+    migen
+  ];
+
+  checkInputs = [
+    numpy
+    microscope
```
Comment (risicle) [inline pkgs/development/python-modules/misoc/default.nix:32]: The package doesn't appear to make any mention of the `microscope` package and the tests pass fine without this here.

File: pkgs/development/python-modules/pyrp3/default.nix (hunk 1 above)
Comment (doronbehar) [reply pkgs/development/python-modules/pyrp3/default.nix:35]: > `pythonImportsCheck` though?

Thanks for the reminder, it pointed out that there were issues in the result.

File: pkgs/development/python-modules/misoc/default.nix (hunk 2 above)
Comment (doronbehar) [reply pkgs/development/python-modules/misoc/default.nix]: > The package doesn't appear to make any mention of the `microscope` package and the tests pass fine without this here.

Thanks!

**Comment by doronbehar** [comment]
> Tests fail for me on macos 10.15. Possibly should mark as broken?

I will try to debug it via ofborg a bit, and if issues will be difficult I'll mark it as broken.

**Comment by doronbehar** [comment]
@ofborg build python3Packages.microscope

**Comment by doronbehar** [comment]
I don't need microscope as a dependency at all, probably I needed it as a dep before an update :sweat_smile: .

**Comment by doronbehar** [comment]
I'm holding off this PR due to the refactoring happening at https://github.com/linien-org/linien/pull/277 .

**Comment by doronbehar** [comment]
Closing in favor of separate PRs.

**Pull request closed without merging**